web/client/plugins/ResourcesCatalog/components/Icon.jsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2024, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState, useEffect } from 'react';
+import { Glyphicon } from 'react-bootstrap';
+import { loadFontAwesome } from '../../../utils/FontUtils';
+import useIsMounted from '../../../hooks/useIsMounted';
+import PropTypes from 'prop-types';
+
+function FaIcon({
+    name,
+    className,
+    style
+}) {
```

Comment:
We can remove this component if we use Glyphicon instead in the other files. Then we can remove the ResourcesCatalog folder